### ISSUE TITLE
Add new blog post

### ISSUE THREAD

**Issue by cjbayesian** [pennsignals/pennsignals.github.io#34]
Post Ideas:
- [ ] Blog idea: Madlibs should def be a blog -- perhaps @shregli to write?
    - Layer in a 'under constraints _______' section.
- [ ] Blog idea: What is data quality in Healthcare?
    - "Can we trust this data?"
    - "Is this data correct?"
        - It is erroneous (wrong value, wrong patient, human entry error)
        - It doesn't mean what you think it means (wrong definition of clinical concept)
        - It was valid once, but is not any mor
- [ ] Blog idea: AUC is a lossy compression of model performance
- [ ] Blog idea: Pre-test probability updating using Sens + Spec

**Comment by shregli** [comment]
Yes I will sign up for writing MadLibs blog post – should prob try to get this published as well.

**Comment by cjbayesian** [comment]
Workaround:
![image](https://user-images.githubusercontent.com/1396669/74063518-932c7680-49be-11ea-872b-22b07022a398.png)

**Comment by cjbayesian** [comment]
![image](https://user-images.githubusercontent.com/1396669/74167638-8b571700-4bf6-11ea-9800-3f6cde955407.png)

**Comment by cjbayesian** [comment]
![image](https://user-images.githubusercontent.com/1396669/74166363-6feb0c80-4bf4-11ea-8f8c-f461a49a8619.png)

**Comment by cjbayesian** [comment]
erroneous bp (inversion):
![image](https://user-images.githubusercontent.com/1396669/74168045-2819b480-4bf7-11ea-86a4-93b2dc8a758c.png)

**Comment by cjbayesian** [comment]
![image](https://user-images.githubusercontent.com/1396669/74168329-8e063c00-4bf7-11ea-96a9-d674ede1b557.png)

**Comment by cjbayesian** [comment]
latency
![image](https://user-images.githubusercontent.com/1396669/74168731-27355280-4bf8-11ea-8ceb-3aa982148139.png)

**Comment by cjbayesian** [comment]
Workarounds:
![image](https://user-images.githubusercontent.com/1396669/74171311-5fd72b00-4bfc-11ea-9ff8-d8b42db18862.png)

**Comment by cjbayesian** [comment]
![image](https://user-images.githubusercontent.com/1396669/74366842-c300c300-4d9e-11ea-8ae3-56c770199ce6.png)

**Comment by cjbayesian** [comment]
![image](https://user-images.githubusercontent.com/1396669/74367479-f2fc9600-4d9f-11ea-9cec-ecfcef7e59eb.png)
